src/Bicep.Core/Configuration/ConfigHelper.cs

```diff
@@ -0,0 +1,133 @@
+// Copyright (c) Microsoft Corporation.
```

Comment:
For later: Bicep files opened in the editor do not update diagnostics if the config file changes.  Also, what if one is added or deleted?

Reply:
Sometimes they do, soimetimes not?

Reply:
This behavior is going to confuse people. If we don't fix it in this PR, we should address it immediately after before 0.4

Reply:
This will require a filewatcher in the folder tree for created/deleted files. Currently the config is reloaded whenever the linter runs on a file.